public/app/features/library-panels/components/ChangeLibraryPanelModal/ChangeLibraryPanelModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ConfirmModal } from '@grafana/ui';
+
+import { PanelModel } from '../../../dashboard/state';
+
+export interface ChangeLibraryPanelModalProps {
+  panel: PanelModel;
+  onConfirm: () => void;
+  onDismiss: () => void;
+}
+
+export const ChangeLibraryPanelModal = ({ onConfirm, onDismiss, panel }: ChangeLibraryPanelModalProps): JSX.Element => {
+  const isLibraryPanel = Boolean(panel.libraryPanel);
```

Comment:
Should we use the `isPanelModelLibraryPanel` guard here?

Reply:
Hmm, yeah I was thinking about that earlier, but I changed it. I'll revert that to isPanelModelLibraryPanel. Thank you for suggesting.

Reply:
Fixed!